src/components/HomePage/AccountCard/AccountCard.tsx

```diff
@@ -12,14 +12,14 @@ const AccountCard = ({
   title,
   amount,
   transactions,
-  onSeeAll,
+  onSeeAll
 }: Props) => {
-  const pending = transactions.filter(
-    (tx) => tx.status === TransactionStatus.PENDING
-  )
-  const latest = transactions.filter(
-    (tx) => tx.status !== TransactionStatus.PENDING
-  )
+  const pending = transactions
+    .filter((tx) => tx.status === TransactionStatus.PENDING)
+    .slice(0, 4)
```

Comment:
What do you think of moving this value to a constant like `MAX_TXS_TO_SHOW_PER_STATUS=4` to the top of the file?

Reply:
done! thanks!